_security-analytics/threat-intelligence/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Threat intelligence
+nav_order: 40
+has_children: true
+---
+
+# Threat intelligence
+
+Threat intelligence in Security Analytics offers the capability to integrate your threat intelligence feeds. Feeds are comprised Indicators of Compromise (IOCs), which search for malicious indicators within your data by setting up a threat intelligence monitor. These monitors generate findings and can send notifications when malicious IPs, domains, or hashes referenced from the threat intelligence feeds match your data.
```

Comment:
I had a different rewrite that looks like was overwritten?

Reply:
Let me look into that.